clappr/clappr-ios
# Events renaming

### Goal 
To deprecate event names and rename/create new

### How to test
1 - Listen to new events and check if they were called
2 - Run ios and tvos tests

#### bonus
fix load bug which was not updating source option

#### ps
wiki will be update on release process
